src/platform/site-wide/announcements/tests/selectors.unit.spec.js

```diff
@@ -44,6 +44,11 @@ describe('selectAnnouncement', () => {
           name: 'dummy6',
           paths: /^(\/some-route-6\/)$/,
         },
+        {
+          name: 'dummy7',
+          paths: /^(\/unique-route\/)$/,
+          expiresAt: '2019-11-11',
```

Comment:
Might be good to have three tests for this:
```js
expiresAt: moment().subtract(1, 'day').format('YYYY-MM-DD'), // Yesterday
```
```js
expiresAt: moment().format('YYYY-MM-DD'), // Today
```
```js
expiresAt: moment().add(1, 'day').format('YYYY-MM-DD'), // Tomorrow
```
That way all the boundaries are tested.